### DISCUSSION TITLE
Welcome to Sorting-Visualizer Discussions!

### DISCUSSION THREAD

**Discussion by techieshouvik** [techieshouvik/Sorting-Visualizer#2] [General]
<h2>## 👋 Welcome Folks!</h2>
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about yourself and how you Code? We would love to hear you!
 Happy Coding 😄

**Comment by AymKh** [comment]
@techieshouvik  couldn't help but notice the "add-license-1" branch......doe it serve any purpose ?

**Reply by techieshouvik** [reply]
At one point it did, but now it doesn't. Good call, we'll remove it right away. :)
